go.mod

```diff
@@ -96,7 +96,7 @@ require (
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.15.0
 	github.com/rancher/aks-operator v1.0.1-rc14
-	github.com/rancher/apiserver v0.0.0-20210519053359-f943376c4b42
+	github.com/rancher/apiserver v0.0.0-20210716201731-fcd925c99ba8
```

Comment:
This change is bringing in something that breaks the UI for non-admin users. UI console error is `Loading error Error: Cannot read property 'startsWith' of undefined`, rancher log error `[ERROR] Error during subscribe websocket: close sent`

Reply:
Yup, it's intentional. UI has some work to do now.